Algorithms/Divide-and-Conquer/Strassens Algorithm.cpp

```diff
@@ -0,0 +1,129 @@
+#include <bits/stdc++.h>
+#include <vector>
+#include<stdio.h>
+#define ll long long int
+using namespace std;
+void sumsub(vector< vector<ll> > &A, vector< vector<ll> > &B, vector< vector<ll> > &C, ll m, char s) {
```

Comment:
Please write appropriate comments what each function does and also follow the naming conventions as mentioned in the [Coding standards wiki page](https://github.com/sahilbansal17/Competitive_Coding/wiki/Coding-Standards)